src/main/java/com/github/vkorobkov/jfixtures/domain/Row.java

```diff
@@ -28,4 +29,24 @@ public Row columns(Map<String, ?> toMerge) {
         );
         return new Row(name, merged);
     }
+
+    public Row columns(Object... keyValuePairs) {
+        int pairsLength = keyValuePairs.length;
+
+        if (pairsLength % 2 != 0) {
+            throw new IllegalArgumentException("Odd number of key/value pairs");
```

Comment:
Can we improve error message a bit? `"parameter <keyValuePairs> is expected to have odd length since it represents key/value pairs"`